exe/Wrapper.hs

```diff
@@ -1,33 +1,24 @@
-{-# LANGUAGE CPP             #-}
 {-# LANGUAGE RecordWildCards #-}
 -- | This module is based on the hie-wrapper.sh script in
 -- https://github.com/alanz/vscode-hie-server
 module Main where
 
-#if __GLASGOW_HASKELL__ < 804
-import           Data.Semigroup
-#endif
-
 import Arguments
--- import Control.Concurrent.Extra
 import Control.Monad.Extra
-import           Data.Foldable
-import           Data.List
--- import Data.List.Extra
--- import qualified Data.Text as T
--- import qualified Data.Text.IO as T
--- import Development.IDE.Types.Logger
+import Data.Foldable
+import Data.List
+import Data.Void
 import HIE.Bios
-import           Ide.Cradle (findLocalCradle)
-import           Ide.Logger (logm)
```

Comment:
Why is the logger gone here?

Reply:
The logger never worked in the first place from what I could tell, since the wrapper didn't call the haskell-lsp setup function! So all those logm/debugm calls seemed to just go down the drain.